app/src/main/java/org/gnucash/android/db/adapter/ScheduledActionDbAdapter.java

```diff
@@ -135,20 +136,19 @@ public long updateRecurrenceAttributes(@NonNull ScheduledAction scheduledAction)
         stmt.bindLong(4, schedxAction.getEndTime());
         stmt.bindLong(5, schedxAction.getLastRunTime());
         stmt.bindLong(6, schedxAction.isEnabled() ? 1 : 0);
-        stmt.bindString(7, schedxAction.getCreatedTimestamp().toString());
+        stmt.bindString(7, TimestampHelper.getUtcStringFromTimestamp(schedxAction.getCreatedTimestamp()));
         if (schedxAction.getTag() == null)
             stmt.bindNull(8);
         else
             stmt.bindString(8, schedxAction.getTag());
-        stmt.bindString(9, Integer.toString(schedxAction.getTotalPlannedExecutionCount()));
+        stmt.bindLong(9, schedxAction.getTotalPlannedExecutionCount());
```

Comment:
This code is changing the type from an integer to a long. Might this cause problems in the database? I haven't seen how the column being written to is defined, so I can't know for sure if this is problematic or not.

Also, this code is in a method that can potentially be used with any/all SQLiteStatements in the application. So it might not be too easy to determine if this is a safe change to make or not.

Reply:
`+ ScheduledActionEntry.COLUMN_TOTAL_FREQUENCY + " integer default 0, "`

Reply:
So does the integer data type in sqlite support longs?